x-pack/plugins/ml/public/application/datavisualizer/file_based/components/results_links/results_links.tsx

```diff
@@ -142,9 +148,26 @@ export const ResultsLinks: FC<Props> = ({
     }
   }
 
+  function openInDiscover(e: React.MouseEvent<HTMLButtonElement>) {
+    e.preventDefault();
+    getApplication().navigateToUrl(discoverLink);
```

Comment:
Should this fallback to something else instead of `''` if `getUrlGenerator(DISCOVER_APP_URL_GENERATOR)` throws an error?

Reply:
Button is not visible if discoverLink is not provided so onClick could not be called.